src/databricks/labs/remorph/helpers/file_utils.py

```diff
@@ -88,3 +88,14 @@ def read_file(filename: str | Path) -> str:
     # pylint: disable=unspecified-encoding
     with Path(filename).open() as file:
         return file.read()
+
+
+def write_file(file_path: str, file_content: str, encoding: str = 'utf-8'):
```

Comment:
remove, we don't need it. use `pathlib.Path.open()` instead

Reply:
Done.